windows/renderer/index.html

```
@@ -0,0 +1,116 @@
+  <body>
+    <div class="shell sidebar-footer-workspace" id="shell">
+      <aside class="sidebar" id="sidebar">
+        <div class="brand">
+          <div class="brand-mark">cm</div>
+          <div>
+            <div class="brand-title">cmux</div>
+            <div class="brand-subtitle">Windows</div>
+          </div>
+        </div>
+        <div class="sidebar-tools">
+          <button class="rail-button is-active" id="workspacesRailButton" title="Workspaces" aria-label="Workspaces">▦</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)